src/components/Service/service.js

```diff
@@ -0,0 +1,74 @@
+import axios from "axios";
+import constant from "components/constant";
+
+const getTasks = async() => {
+  try {
+    const resp = await axios.get(`${constant}/allTasks`);
```

Comment:
You should only have this line in services:
`const resp = await axios.get(`${constant}/allTasks`);`
that is, you take out only the part where the request occurs and that's it

you don't need try/catch here
The request is already in it when you call the function in the components